recipe/meta.yaml

```diff
@@ -23,42 +20,42 @@ requirements:
     - setuptools
     - pip
     - wheel
+    - flit-core <4
 
   run:
     - python
-    - nltk >=3.1
+    - nltk >=3.8
 
-{% set tests_to_ignore = "--ignore=tests/tests/test_blob.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_classifiers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_np_extractor.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_tokenizers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_sentiments.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_taggers.py" %}
 
 test:
   source_files:
     - tests
+    - pyproject.toml
   imports:
     - textblob
     - textblob.en
     - textblob.unicodecsv
   commands:
     - pip check
-    - pytest -v tests {{ tests_to_ignore }}
+    # Downloading nltk assets for testing. textblob.download corpora is insufficient for testing needs.
+    - python -c "import nltk; nltk.download('punkt_tab')"
+    - python -c "import nltk; nltk.download('averaged_perceptron_tagger_eng')"
+    - python -m textblob.download_corpora
```

Comment:
I'm not a big fan of downloading data on the fly like this. Do you think we could somehow pre-download the data or maybe avoid all the tests that need the downloaded files? I'm not familiar with these packages, so maybe that's not possible...

Reply:
We don't have to do these tests if we don't want to.  I could just skip the ones that require data.  The good thing is that it only downloads if it isn't there so subsequent builds on the same machine won't download the data again.

Reply:
These tests do test the compatibility with nltk though.  So if nltk ever creates a breaking change these will pick them up.

Reply:
I will remove for now but leave the comments in so people can test easily.

Reply:
Thanks for taking some time to take a look at it. My main concerns with downloading things like this is that we don't have an easy way to know that we can really trust what's being download or even know what's being downloaded. The is also other concerns that relate to how we operate that I'm not gonna go over here but can definitely follow up in private.